applications/PoromechanicsApplication/custom_strategies/strategies/poromechanics_newton_raphson_nonlocal_strategy.hpp

```diff
@@ -104,7 +104,7 @@ class PoromechanicsNewtonRaphsonNonlocalStrategy : public PoromechanicsNewtonRap
                     mpNonlocalDamageUtility = new NonlocalDamage3DUtilities();
```

Comment:
are you checking 2D but setting 3D

Reply:
You were looking at two non-consecutive lines

Reply:
If it’s 2D, then I set 2D utility. The 3D is in the else part